releasenotes/notes/0.20/upgrade-convert-scheduling-passes-to-analysis-04333b6fef524d21.yaml

```diff
@@ -1,25 +1,24 @@
 ---
 features:
   - |
-    New transpiler pass :class:`~qiskit.transpiler.passes.scheduling.padding.PadDelay`
-    has been added. This pass fills idle time on the qubit wire with ``Delay`` instruction.
-    The input DAG should be scheduled in advance.
+    A new transpiler pass :class:`.PadDelay` has been added. This pass fills
+    idle time on the qubit wires with :class:`~.circuit.Delay` instructions.  The input
+    DAG should be scheduled in advance.
 upgrade:
   - |
-    Scheduling passes :class:`~qiskit.transpiler.passes.scheduling.asap.ASAPSchedule` and
-    :class:`~qiskit.transpiler.passes.scheduling.alap.ALAPSchedule` have been turned into
-    analysis pass that does not modify the input DAG circuit.
-    Now these passes just analyze the time slot allocation of every ``DAGOpNode``s and
+    Scheduling passes :class:`.ASAPSchedule` and :class:`.ALAPSchedule` have
```

Comment:
This release note and the others about the scheduling passes I completely rewrote in https://github.com/Qiskit/qiskit-terra/pull/7835 so there will probably be a merge conflict here when I update the release note branch after #7835 merges. But I'll handle that when the time comes (and try to factor this in as part of that fix).

Reply:
This particular one isn't in that PR (though maybe it should be?) - I had originally changed all those release notes, then I realised and reverted the changes to try and minimise conflicts.